<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentManagerMainWindow</class>
 <widget class="QMainWindow" name="RentManagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rent Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/cottage.png</normaloff>:/images/cottage.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Payment Calendar</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>208</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Companies/Properties/Units</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="trvBrowser">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Year:</string>
               </property>
              </widget>
              <widget class="QComboBox" name="cboYear">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>2014</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2015</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2016</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2017</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2018</string>
                </property>
               </item>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="txtCalendar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>53</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>0</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Reports</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="lstReports">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Contact List</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPrintPreviewWidget" name="printPreviewWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionCompanies"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionUnites"/>
    <addaction name="actionTenants"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAccounting">
    <property name="title">
     <string>Acco&amp;unting</string>
    </property>
    <addaction name="actionView_Tenan_Accounts"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Invoice"/>
    <addaction name="actionMulti_Invoicing"/>
    <addaction name="separator"/>
    <addaction name="actionReceive_Payments"/>
    <addaction name="actionView_Payments"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>&amp;Management</string>
    </property>
    <addaction name="actionAssign_Unit_To_Tenant"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>&amp;Reports</string>
    </property>
    <addaction name="actionCompanies_Properties"/>
    <addaction name="actionAll_Units"/>
    <addaction name="actionVacan_Units"/>
    <addaction name="actionOccupied_Units"/>
    <addaction name="separator"/>
    <addaction name="actionContact_List"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManagement"/>
   <addaction name="menuAccounting"/>
   <addaction name="menuReports"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCompanies"/>
   <addaction name="actionProperties"/>
   <addaction name="separator"/>
   <addaction name="actionTenants"/>
   <addaction name="separator"/>
   <addaction name="actionMulti_Invoicing"/>
   <addaction name="actionReceive_Payments"/>
  </widget>
  <action name="actionCompanies">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/company-building-icon.png</normaloff>:/images/company-building-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Companies</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Home-icon.png</normaloff>:/images/Home-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Properties</string>
   </property>
  </action>
  <action name="actionUnites">
   <property name="text">
    <string>&amp;Units</string>
   </property>
  </action>
  <action name="actionTenants">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Preppy-icon.png</normaloff>:/images/Preppy-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Tenants</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionView_Tenan_Accounts">
   <property name="text">
    <string>&amp;View Tenan Accounts</string>
   </property>
  </action>
  <action name="actionCreate_Invoice">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/invoice-icon.png</normaloff>:/images/invoice-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create Invoice</string>
   </property>
  </action>
  <action name="actionMulti_Invoicing">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/invoice-icon.png</normaloff>:/images/invoice-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Multi Invoicing</string>
   </property>
  </action>
  <action name="actionReceive_Payments">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Sales-by-payment-method-icon.png</normaloff>:/images/Sales-by-payment-method-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Receive Payments</string>
   </property>
  </action>
  <action name="actionAssign_Unit_To_Tenant">
   <property name="text">
    <string>&amp;Assign Unit To Tenant</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Files-New-File-icon.png</normaloff>:/images/Files-New-File-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open-file-icon.png</normaloff>:/images/open-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Save-icon.png</normaloff>:/images/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionView_Payments">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Sales-by-payment-method-icon.png</normaloff>:/images/Sales-by-payment-method-icon.png</iconset>
   </property>
   <property name="text">
    <string>View &amp;Payments</string>
   </property>
  </action>
  <action name="actionCompanies_Properties">
   <property name="text">
    <string>&amp;Companies / Properties</string>
   </property>
  </action>
  <action name="actionAll_Units">
   <property name="text">
    <string>&amp;All Units</string>
   </property>
  </action>
  <action name="actionVacan_Units">
   <property name="text">
    <string>&amp;Vacant Units</string>
   </property>
  </action>
  <action name="actionOccupied_Units">
   <property name="text">
    <string>&amp;Occupied Units</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionContact_List">
   <property name="text">
    <string>Contact &amp;List</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPrintPreviewWidget</class>
   <extends>QWidget</extends>
   <header>qprintpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
